### PR TITLE
feat(table) implements count command

### PR DESCRIPTION
**Pull request by youen**

### CODE REVIEW

**Comment by youen** [comment]
output example :

```
lino table count source                             
inventory: 4581
payment: 16049
rental: 16044
staff: 2
actor: 200
customer: 599
film: 1000
film_category: 1000
store: 2
language: 6
category: 16
city: 600
film_actor: 5462
address: 603
country: 109
```

tables are not sort by name or size and list is randomized.

File: internal/app/table/count.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+			for _, tableCount := range result {
+				fmt.Printf("%s: %d\n", tableCount.Table.Name, tableCount.Count)
+			}
+		},
+	}
+	cmd.SetOut(out)
+	cmd.SetErr(err)
+	cmd.SetIn(in)
+	cmd.Flags().StringVarP(&exportType, "export", "e", "", "export type for the column")
+	cmd.Flags().StringVarP(&importType, "import", "i", "", "import type for the column")
```
Comment (adrienaury) [inline internal/app/table/count.go:78]: Don't need these flags :)

File: internal/app/table/count.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	"github.com/cgi-fr/lino/pkg/dataconnector"
+
+	"github.com/cgi-fr/lino/pkg/table"
+	"github.com/spf13/cobra"
+)
+
+// newCountCommand implements the cli table count command
+func newCountCommand(fullName string, err *os.File, out *os.File, in *os.File) *cobra.Command {
+	// local flags
+	var exportType, importType string
```
Comment (adrienaury) [inline internal/app/table/count.go:34]: Don't need these flags :)

**Comment by adrienaury** [comment]
Lino didn't catch any HTTP 404 in my test

```console
❯ curl http://:@http-connector:8080/api/v1/data/actor/count
{"timestamp":"2022-02-25T15:09:59.296+00:00","status":404,"error":"Not Found","path":"/api/v1/data/actor/count"
❯ lino table count http -v5
4:07PM INF Logger level set to trace
4:07PM INF Start LINO color=auto debug=false log-json=false verbosity=5
4:07PM DBG External connector request url=http://:@http-connector:8080/api/v1/data/actor/count
4:07PM DBG External connector request url=http://:@http-connector:8080/api/v1/data/address/count
4:07PM DBG External connector request url=http://:@http-connector:8080/api/v1/data/category/count
4:07PM DBG External connector request url=http://:@http-connector:8080/api/v1/data/city/count
4:07PM DBG External connector request url=http://:@http-connector:8080/api/v1/data/country/count
4:07PM DBG External connector request url=http://:@http-connector:8080/api/v1/data/customer/count
4:07PM DBG External connector request url=http://:@http-connector:8080/api/v1/data/film/count
4:07PM DBG External connector request url=http://:@http-connector:8080/api/v1/data/film_actor/count
4:07PM DBG External connector request url=http://:@http-connector:8080/api/v1/data/film_category/count
4:07PM DBG External connector request url=http://:@http-connector:8080/api/v1/data/inventory/count
4:07PM DBG External connector request url=http://:@http-connector:8080/api/v1/data/language/count
4:07PM DBG External connector request url=http://:@http-connector:8080/api/v1/data/payment/count
4:07PM DBG External connector request url=http://:@http-connector:8080/api/v1/data/rental/count
4:07PM DBG External connector request url=http://:@http-connector:8080/api/v1/data/staff/count
4:07PM DBG External connector request url=http://:@http-connector:8080/api/v1/data/store/count
actor: 0
address: 0
category: 0
city: 0
country: 0
customer: 0
film: 0
film_actor: 0
film_category: 0
inventory: 0
language: 0
payment: 0
rental: 0
staff: 0
store: 0
4:07PM INF End LINO return=0
```

File: internal/infra/table/http_extractor.go [hunk 3]
```diff
@@ -77,6 +77,42 @@ func (e *HTTPExtractor) Extract() ([]table.Table, *table.Error) {
...
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return 0, &table.Error{Description: err.Error()}
+	}
+	defer resp.Body.Close()
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return 0, &table.Error{Description: err.Error()}
+	}
+	container := struct {
```
Comment (adrienaury) [inline internal/infra/table/http_extractor.go:102]: You should check status code

```go
	if code < 200 || code >= 300 {
		return &push.Error{Description: fmt.Sprintf("HTTP request returned status code %d", code)}
	}
```

**Review by adrienaury — CHANGES_REQUESTED** [review]

**Comment by youen** [comment]
should we upgrade version's api to v2 ?

**Comment by adrienaury** [comment]
> should we upgrade version's api to v2 ?

No, there is no changes in the api, only added feature

**Review by adrienaury — APPROVED** [review]

**Pull request merged**